automated-tests/src/test/resources/cucumber/network/fcoeNetwork.feature

```diff
@@ -47,6 +47,17 @@ Feature: In order to manage FCoE Networks
       And OneView gets Resource by ID
     Then I get a Resource Name
 
+  @patch
+  Scenario: Update an FcoE Network by Patch
+    Given name "fcoe-network-bdd" for Resource
+      And Resource values will be updated as follows:
+      | op    | replace      |
+      | path  | /scopeUris   |
+      | value | Sample Scope |
+    When OneView gets Resource by Name
+      And OneView runs FcoE Network patch
+    Then Resource is found
```

Comment:
I think we should change the verification for a success status, or to verify the resources properties (scopeUris attribute).